### PR TITLE
Style page wells

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-43

~~*NOTE:* This is based on work in #1163, so this PR will have less files changed once the other is merged.~~

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

local-styles-wells

### CODE REVIEW

File: demo/assets/_code.less [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+/* Remove background color from syntax highlighted code (prism.css) */
+pre[class*="language-"] {
+    &, code {
+        background: transparent;
+        border: 0;
+    }
+}
+
+/* Custom Coloring (outside of prism syntax-highlighting) */
```
Comment (CITguy) [inline demo/assets/_code.less]: *NOTE:* The previous namespaces (`.component-description, .component-demo, .markdown`) were unnecessary so long as we override background and border for the syntax-highlighted sections (style block above).

File: demo/assets/_code.less [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/* Code Styles */
+
+/* Remove background color from syntax highlighted code (prism.css) */
+pre[class*="language-"] {
+    &, code {
```
Comment (CITguy) [inline demo/assets/_code.less]: Because of the higher specificity, this style will override our custom styles below.

**Comment by lfielder** [comment]
Design - LGTM

**Comment by parlarjb** [comment]
LGTM

File: demo/styles/wells.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<rx-page title="'Wells'" class="styleguide">
+  <h3>Page Wells</h3>
+  <p>
+    The <code>.well</code> class creates a well-effect--a box with significant
```
Comment (bry4n) [inline demo/styles/wells.html]: `well-effect--a` ?

Comment (CITguy) [inline demo/styles/wells.html]: Ah, yes. HTML doesn't automatically change `--` to an emdash. I'll update.

**Comment by CITguy** [comment]
screenshot updated

**Comment by bry4n** [comment]
LGTM

**Pull request merged**